clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -398,6 +399,7 @@ export default async function handler(
       : ""
   }
   window.Fides.config = ${fidesConfigJSON};
+  window.Fides.release_version = "${releaseVersion}";
```

Comment:
nit: I'd prefer it to be `window.Fides.version` 👍 

But also, we shouldn't be injecting that in the privacy-center bundle, it should already be there in the FidesJS file, sorry!